test/integration/discovered_hosts_test.rb

```diff
@@ -0,0 +1,158 @@
+require 'test_plugin_helper'
+require 'integration_test_helper'
+
+class DiscoveredHostsTest < IntegrationTestWithJavascript
+  let(:discovered_hosts) { Host::Discovered.all }
+
+  setup do
+    set_default_settings
+    @discovered_host = FactoryGirl.create(:discovered_host, :with_facts)
```

Comment:
Our API explicitly creates taxonomy for discovered hosts, I was wondering if you could move the taxonomy let blocks here and set it explicitly here as well.